yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -256,16 +257,26 @@ export class PXEOracleInterface implements ExecutionDataProvider {
    * Fetches the current chain id.
    * @returns The chain id.
    */
-  public async getChainId(): Promise<number> {
-    return await this.aztecNode.getChainId();
+  public getChainId(): Promise<number> {
+    return this.aztecNode.getChainId();
```

Comment:
Pulled the Jan here 😜

It bother me to have there the unnecessary async and await.